Library/Homebrew/cask/url.rb

```diff
@@ -158,9 +162,8 @@ def call
       sig {
         params(
           uri:   T.any(URI::Generic, String),
-          block: T.proc.params(arg0: T.all(String, PageWithURL))
-                       .returns(T.any(T.any(URI::Generic, String), [T.any(URI::Generic, String), Hash])),
```

Comment:
Note the slight change in `BlockReturn` – unpacking what are believe is a redundant, nested `T.any`